WordPressAuthenticator/Signin/LoginPrologueViewController.swift

```diff
@@ -138,41 +150,35 @@ class LoginPrologueViewController: LoginViewController {
         storedCredentialsAuthenticator.showPicker(from: navigationController)
     }
 
-    // MARK: - Segue
-
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        super.prepare(for: segue, sender: sender)
-
-        if let vc = segue.destination as? NUXButtonViewController {
-            buttonViewController = vc
-        }
-    }
-
     private func configureButtonVC() {
```

Comment:
Can we maybe rename this method because it sounds similar to `configureButtonViewController`? Maybe `buildButtons`? Plus, I wonder why this method has to be called in `viewWillAppear`, but if we have no idea why, we should probably keep that as-is for safety reasons.

Reply:
I decided to renamed the `configureButtonViewController` method in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/695/commits/47b8f3c68b723703c7be651ee9bf337e6af64c9b